client/modules/IDE/components/Header/MobileNav.jsx

```diff
@@ -0,0 +1,338 @@
+          }
+        }
+      }
+    }
+  }
+`;
+
+const MobileNav = (props) => {
+  const project = useSelector((state) => state.project);
+  const user = useSelector((state) => state.user);
+
+  const { t } = useTranslation();
+
+  const { pathname } = useLocation();
+  const editorLink = useSelector(selectSketchPath);
+
+  // TODO: remove the switch and use a props like mobileTitle <Nav layout=“dashboard” mobileTitle={t(‘Login’)} />
+  function resolveTitle() {
+    switch (pathname) {
+      case '/':
+        return project.name;
+      case '/login':
+        return t('LoginView.Login');
+      case '/signup':
+        return t('LoginView.SignUp');
+      case '/account':
+        return t('AccountView.Settings');
+      case '/p5/sketches':
+      case '/p5/collections':
+        return t('Nav.File.Examples');
+      case `/${user.username}/assets`:
+      case `/${user.username}/collections`:
+      case `/${user.username}/sketches`:
+        return 'My Stuff';
+      default:
+        return project.name;
+    }
+  }
+
+  const title = useMemo(resolveTitle, [project, pathname]);
+
+  const Logo = AsteriskIcon;
+  return (
+    <Nav>
+      <LogoContainer>
+        <Logo />
+      </LogoContainer>
+      <Title>
+        <h1>{title}</h1>
+        {project?.owner && title === project.name && (
+          <h5>by {project?.owner?.username}</h5>
+        )}
+      </Title>
+      <Options>
+        {user.authenticated ? (
+          <AccountMenu />
+        ) : (
+          <div>
+            <Link to="/login">
+              <IconButton icon={AccountIcon} />
+            </Link>
+          </div>
+        )}
+        {title === project.name ? (
+          <MoreMenu />
+        ) : (
+          <div>
+            <Link to={editorLink}>
+              <IconButton icon={EditorIcon} />
+            </Link>
+          </div>
+        )}
+      </Options>
+    </Nav>
+  );
+};
+
+const AccountMenu = () => {
+  const user = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+
+  return (
+    <div>
+      <IconButton icon={AccountIcon} />
+      <ul>
+        <li className="user">{user.username}</li>
+        <li>
+          <Link to={`/${user.username}/sketches`}>
+            <button>My Stuff</button>
+          </Link>
+        </li>
+        <li>
+          <Link to="/account">
+            <button>Settings</button>
+          </Link>
+        </li>
+        <li>
+          <button onClick={() => dispatch(logoutUser())}>Log Out</button>
+        </li>
+      </ul>
+    </div>
+  );
+};
+
+const MoreMenu = () => {
+  // TODO: use selectRootFile selector
+  const rootFile = useSelector(
+    (state) => state.files.filter((file) => file.name === 'root')[0]
+  );
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { newSketch, saveSketch } = useSketchActions();
+
+  const cmRef = useContext(CmControllerContext);
+
+  return (
+    <div>
+      <IconButton icon={MoreIcon} />
```

Comment:
Can we add a `transform: rotate(90deg);` to this icon to make the dots vertical?

Reply:
I updated the svg